pkg/commands/create/create.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package create
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kustomize/v3/k8sdeps/kunstruct"
+	"sigs.k8s.io/kustomize/v3/pkg/commands/kustfile"
+	"sigs.k8s.io/kustomize/v3/pkg/commands/util"
+	"sigs.k8s.io/kustomize/v3/pkg/fs"
+	"sigs.k8s.io/kustomize/v3/pkg/pgmconfig"
+)
+
+type createFlags struct {
+	resources       []string
+	namespace       string
+	annotations     string
+	labels          string
+	prefix          string
+	suffix          string
+	detectResources bool
+	detectRecursive bool
+	path            string
```

Comment:
Do you plan to add a flag for `path` as well? So that user can specify `--dir <dir>` instead of using current directory.

Reply:
If I understand this correctly, current change is to add a kustomization.yaml to an existing directory. How about creating a new kustomization directory without touching the existing directory? The kustomization.yaml is written to that directory and the resource files are copied to that directory.

Reply:
> Do you plan to add a flag for path as well?

I initially added the path field to the struct to simplify testing. Didn't want to have to also implement relative path handling in FakeFS. 

> If I understand this correctly, current change is to add a kustomization.yaml to an existing directory.

That's correct, current change only targets creating a new kustomization file in the current directory.

> The kustomization.yaml is written to that directory and the resource files are copied to that directory.

Thought about including this but opted to limit the scope of this PR. I can add that as a followup PR or would you prefer it to be included in this one?

Reply:
@Liujingfang1 Thoughts on above? ☝️

Reply:
fwiw, i like the simplicity of creating in the current working directory.  a surrounding script can cd to the right place beforehand, or copy it afterwards.  Like also that this fails if a kustomization file already exists.

in v0.1 or something we had a create (or init) command that didn't have any flags to add data and thus made little sense.